gulpfile.js

```diff
@@ -14,6 +14,8 @@ plugins.sass = require('gulp-sass');
 plugins.gulpStylelint = require('gulp-stylelint');
 plugins.gulpif = require('gulp-if');
 plugins.postcss = require('gulp-postcss');
+plugins.hash = require('gulp-hash-filename');
```

Comment:
This is subtle: we're now relying on the hash generated by this library being the same as the one we're generating in Python. While I can [see that it is](https://github.com/intervalia/gulp-hash-filename/blob/master/lib/generateHash.js), it's worth at least a code comment to explain the two need to be kept in sync.